.github/workflows/build-push-docker-images.yml

```
@@ -40,9 +45,41 @@ jobs:
           ref: ${{ inputs.ref }}
           fetch-depth: 1
           persist-credentials: false
-          sparse-checkout: |
-            ${{ inputs.context }}
-          sparse-checkout-cone-mode: true
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v3
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)